### DISCUSSION TITLE
Dumb Question - HyperSPI on Mac Mini?

### DISCUSSION THREAD

**Discussion by wilhelmthorpe** [awawa-dev/HyperSPI#38] [Q&A] [answered]
Can HyperSPI be used with a mac mini setup or only with a Raspberry Pi?

**Comment by awawa-dev** [comment, marked as answer]
Ony RPi or other SBC that have hardware SPI interface and it is supported by the OS (Linux)

**Reply by wilhelmthorpe** [reply]
@awawa-dev Ah, okay. Thank you!

**Reply by wilhelmthorpe** [reply]
@awawa-dev I've been wanting an excuse to get an RPi anyway! I ordered the Model 3 B+
